src/build123d/exporters3d.py

```diff
@@ -153,25 +153,27 @@ def _create_xde(to_export: Shape, unit: Unit = Unit.MM) -> TDocStd_Document:
     return doc
 
 
-def export_brep(to_export: Shape, file_path: Union[str, BytesIO]) -> bool:
+def export_brep(
```

Comment:
`BytesIO` provides a mechanism for in memory data exchange which seems to be dropped here, or at least not visible.

Reply:
Pushed a fix and added tests to make sure the next guy doesn't accidentally clobber that behavior too :sweat_smile: